pysaliency/dataset_config.py

```diff
@@ -42,6 +46,10 @@ def apply_dataset_filter_config(stimuli, fixations, filter_config):
         'train_split': train_split,
         'validation_split': validation_split,
         'test_split': test_split,
+        'filter_scanpaths_by_attribute': add_stimuli_argument(filter_scanpaths_by_attribute),
+        'filter_fixations_by_attribute': add_stimuli_argument(filter_fixations_by_attribute),
+        'filter_stimuli_by_attribute': filter_stimuli_by_attribute,
+        'filter_scanpaths_by_lengths': add_stimuli_argument(filter_scanpaths_by_lengths)
```

Comment:
I just noticed that technically, the function and the key here should be called `filter_scanpaths_by_length`. Could you change this in all relevant files? Then this PR is good for merging from my side.

Reply:
fixed